docs(load): clarify universal and server load (#11458)
sveltejs/kit: documentation/docs/20-core-concepts/20-load.md

By default, universal `load` functions run on the server during SSR when the user first visits your page. They will then run again during hydration, reusing any responses from [fetch requests](#making-fetch-requests). All subsequent invocations of universal `load` functions happen in the browser. You can customize the behavior through [page options](page-options). If you disable [server side rendering](page-options#ssr), you'll get an SPA and universal `load` functions _always_ run on the client.

> If a route contains both a universal `load` and server `load`, the server `load` will always run first; then the universal `load` runs immediately after.

Reviewer: 'immediately' isn't correct — server `load` function return values are batched into a single response, so it's possible for there to be a gap between one particular `load` function returning and its universal counterpart starting

```suggestion
If a route contains both universal and server `load` functions, the server `load` runs first.
```
